_posts/2021-02-19-community-call-education.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: February Community Call Summary
+tags: [community call]
+posted_by: Julia Damerow, Chris Hill
+---
+
+The February US-RSE Community Call focused on the topic of RSE education and training. It was well attended with about 25 participants. After some updates from the steering committee we started out with a few warm up Mentimeter questions. Turns out Star Wars created a lot of favorite fictional characters (surprise ;) ). It got interesting with the question if there should be some kind of formal RSE training. While 12 people thought there should be, 11 people answered that it depends, while one person didn’t think there should be formal training. This might warrant a follow up discussion on the question of under what circumstances a formal RSE training would be useful or even desired.
```

Comment:
```suggestion
The February US-RSE Community Call focused on the topic of RSE education and training. It was well attended with about 25 participants. After some updates from the steering committee we started out with a few warm up Mentimeter questions. Turns out Star Wars created a lot of favorite fictional characters (surprise ;) ). It got interesting with the question if there should be some kind of formal RSE training. While 12 people thought there should be, 11 people answered that it depends, while one person didn’t think there should be formal training. This might warrant a follow-up discussion on the question of under what circumstances a formal RSE training would be useful or even desired.
```